src/services/job-processor/service.ts

```diff
@@ -91,6 +91,10 @@ export const makeJobProcessor = ({
       }
     }
 
+    logger.info('Updating exit messages left metrics from contract state')
+    const lastRequestedValIx = await executionApi.lastRequestedValidatorIndex()
```

Comment:
I think we need to add error handling here

Reply:
Good catch! Fixed